runtime/src/accounts_background_service.rs

```diff
@@ -91,6 +91,13 @@ impl SnapshotRequestHandler {
                     status_cache_slot_deltas,
                 } = snapshot_request;
 
+                let previous_hash = if test_hash_calculation {
+                    // cannot calculate or verify here because cache has not been flushed and stores don't match
```

Comment:
I don't get the comment.. does it just apply to the non-index case?

Reply:
yes, it just applies to the non-index case. I can clarify the comment.

Reply:
updated the comment